cplusplus/UnconvertFromZDW.cpp

```diff
@@ -216,11 +216,10 @@ UnconvertFromZDW_Base::UnconvertFromZDW_Base(const string &fileName,
 			string cmd;
 			const size_t len = inFileName.size();
 			if (len >= 4 && !strcmp(inFileName.c_str() + len - 3, ".gz")) {
-				//Internal uncompression of .gz files.
-				input = new BufferedInput(inFileName.c_str(), 16*1024, true);
-				return;
-			}
-			if (len >= 5 && !strcmp(inFileName.c_str() + len - 4, ".bz2")) {
+				cmd = "zcat ";
+				cmd += inFileName;
+				cmd.append(" 2>/dev/null"); //we don't need to see any chatter -- we output all relevant error codes ourselves
```

Comment:
With this line in both gz and bz2, I'm left wondering if it shouldn't also be added to xz?

Reply:
zcat and bz2 need it, but I've never seen a need for it with xz.  That said, I can't see how it would hurt to add it.